src/app/modules/form/public-form/public-form.middlewares.ts

```diff
@@ -1,9 +1,14 @@
 // TODO #4279: Remove after React rollout is complete
+import { get } from 'lodash'
+
 import { reactMigration } from '../../../config/config'
 import { ControllerHandler } from '../../core/core.types'
 
 export const injectFeedbackFormUrl: ControllerHandler = (req, res, next) => {
-  const formId = reactMigration.reactToAngularFeedbackFormId
+  const formId =
+    get(req.query, 'view') === 'admin'
+      ? reactMigration.adminSwitchEnvFeedbackFormId
+      : reactMigration.publicSwitchEnvFeedbackFormId
```

Comment:
nit: should we add a joi validation step now that we look at the query? not a big deal since we ignore all values except `admin`.. but still good to have? if we also ignore all values except `admin`, why are we assigning the value of `public` from the frontend? can just be undefined in that case haha

Reply:
I think it's okay, since soon we'll be getting rid of the public form anyway and always default to the single admin form. Then we can also get rid of the query param and just serve the single admin form id. What do you think?